### PR TITLE
Add helm chart and instruction to use

### PR DESCRIPTION
**Pull request by mallow111**
This is to fix issue https://github.com/salesforce/sloop/issues/19

### CODE REVIEW

File: sloop-helm-charts/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Use sloop-helm-charts to deploy your service
+1. Build image and tag, and replace the current ones in values.yaml, we are working an image for people to use, it will be release in a few days.
+2. Create a namespace in your cluster for sloop to run, for example: `kubectl create namespace sloop `
```
Comment (thomashargrove) [inline sloop-helm-charts/README.md]: nit: * only if you dont already have a namespace

File: sloop-helm-charts/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Use sloop-helm-charts to deploy your service
```
Comment (thomashargrove) [inline sloop-helm-charts/README.md]: This is instructions for folks not using tiller (or Spinnaker)

File: sloop-helm-charts/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+1. Build image and tag, and replace the current ones in values.yaml, we are working an image for people to use, it will be release in a few days.
+2. Create a namespace in your cluster for sloop to run, for example: `kubectl create namespace sloop `
+3. Validate your helm chart in local to make sure there is no mistake:`helm template .`
+4. Write to yamil file: `helm template . --namespace sloop> sloop-test.yaml`
+5. Apply the yaml file in your cluster: `kubectl -n sloop apply -f sloop-test.yaml`
+6.  Check if the service is running: kubectl -n sloop get pods
+7.  Once the pod is running, you can use it by:  `kc-aws port-forward -n sloop statefulset/sloop 8080 8000`
```
Comment (thomashargrove) [inline sloop-helm-charts/README.md]: I think this is very rare that folks are running fully locked down clusters, and those folks would already know how to port forward.  At minimum preface this.

**Comment by thomashargrove** [comment]
Dont think we need `sloop-helm-charts/templates/_helpers.tpl` do we?

**Comment by thomashargrove** [comment]
Should we just name the folder `helm`?

File: sloop-helm-charts/README.md (hunk 1 above)
Comment (mallow111) [reply sloop-helm-charts/README.md]: our current helm chart use namespace is default, if people dont change it, they will use default namespace

File: sloop-helm-charts/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
```
Comment (jeunii) [inline sloop-helm-charts/Chart.yaml]: probably would want to update this and provide more description.

File: sloop-helm-charts/templates/statefulset.yaml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+        name: sloop-data
+      - configMap:
+          name: sloop
+        name: sloopconfig
+      serviceAccountName: {{ .Values.serviceAccountName }}
+  volumeClaimTemplates:
+  - metadata:
+     name: sloop-data
+    spec:
+     accessModes: [ "ReadWriteOnce" ]
```
Comment (jeunii) [inline sloop-helm-charts/templates/statefulset.yaml]: Would like to see this come from `values.yaml`.

File: sloop-helm-charts/Chart.yaml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
```
Comment (thomashargrove) [inline sloop-helm-charts/Chart.yaml]: 0.2

Comment (thomashargrove) [reply sloop-helm-charts/Chart.yaml]: actually, what is appVersion vs version?

Comment (mallow111) [reply sloop-helm-charts/Chart.yaml]: appVersion: The version of the app that this contains (optional). This needn't be SemVer.
https://github.com/helm/helm/blob/master/docs/charts.md#the-appversion-field

**Pull request merged**